### PR TITLE
feat: Auth UI

### PR DESCRIPTION
**Pull request by ganesh-kanchi**
Added Login, Logout, and Signup page UI without the functionality.

### CODE REVIEW

**Review by Kothalkarpriya — APPROVED** [review]
Code is clean and good

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,26 @@
 import "./App.css";
 import { Home } from "./pages/Home/Home";
+import {Routes, Route} from "react-router-dom"
+import { Navbar } from "./components/Navbar/Navbar";
+import { Footer } from "./components/Footer/Footer";
+import { ProductsPage } from "./pages/ProductsPage/ProductsPage";
+import { Login } from "./pages/auth/Login";
```
Comment (rushikesh3299) [inline src/App.js]: You can make index file in folders & export from there

File: src/components/Navbar/Navbar.css [hunk 2]
```diff
@@ -18,4 +32,36 @@
...
+}
+
+@media screen and (max-width: 700px){
+  .nav-links *{
+    margin: 0.5rem 0.25rem 0.5rem 0.25rem;
+  }
+
+  .nav-bar .brand-tag{
+    margin: 0 auto 0 0 ;
+    padding-bottom: 0.25rem;
```
Comment (rushikesh3299) [inline src/components/Navbar/Navbar.css]: for small css resultant values <10px using  px is suggested

**Review by rushikesh3299 — APPROVED** [review]
Overall well structured code

File: src/App.js (hunk 1 above)
Comment (ganesh-kanchi) [reply src/App.js]: ok will do, thanks for the review.

File: src/components/Navbar/Navbar.css (hunk 2 above)
Comment (ganesh-kanchi) [reply src/components/Navbar/Navbar.css]: ok, will change it.

**Pull request merged**